src/views/AuthoritiesSearch/Reports/Reports.js

```diff
@@ -0,0 +1,69 @@
+import {
+  FormattedMessage,
+  useIntl,
+} from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { useStripes } from '@folio/stripes/core';
+import {
+  Button,
+  Icon,
+  MenuSection,
+} from '@folio/stripes/components';
+
+import css from './Reports.css';
+
+const Reports = ({
+  onToggle,
+}) => {
+  const stripes = useStripes();
+  const intl = useIntl();
+
+  const showReports = stripes.hasPerm('ui-marc-authorities.authority-record.view') &&
+    stripes.hasPerm('ui-quick-marc.quick-marc-authorities-editor.all') &&
+    stripes.hasPerm('ui-inventory.all-permissions.TEMPORARY') &&
+    (stripes.hasPerm('ui-quick-marc.quick-marc-editor.view') || stripes.hasPerm('ui-quick-marc.quick-marc-editor.all')) &&
+    (stripes.hasPerm('ui-export-manager.export-manager.all') || stripes.hasPerm('ui-export-manager.jobs.downloadAndResend'));
+
+  if (!showReports) return null;
+
+  const renderReport = ({ translationId, onClick }) => (
+    <Button
+      buttonStyle="dropdownItem"
+      onClick={() => {
+        onClick?.();
```

Comment:
Please set `noop` as default value in function args instead of `?.` syntax - it's more common